src/components/BackOffice/Activities/ActivitiesContainer.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import Activities from './Activities'
+
+const activities = [
+
+  {
```

Comment:
Hola Grego, 

Algun motivo por el cual no estas trayendo la info del back todavia?

Reply:
Hola Fedeee! jajaj 

no existe el endpoint aun. El jira card dice que si no hay endpoint crea un hardcode array of objects.